src/nvim/menu.c

```diff
@@ -1360,61 +1329,6 @@ void ex_emenu(exarg_T *eap)
     EMSG2(_("E335: Menu not defined for %s mode"), mode);
 }
 
-#if defined(FEAT_GUI_MSWIN) \
-  || defined(FEAT_GUI_GTK) \
-  || defined(FEAT_BEVAL_TIP)
-/*
- * Given a menu descriptor, e.g. "File.New", find it in the menu hierarchy.
- */
-vimmenu_T *gui_find_menu(char_u *path_name)
-{
-  vimmenu_T   *menu = NULL;
-  char_u      *name;
-  char_u      *saved_name;
-  char_u      *p;
-
-  menu = root_menu;
-
-  saved_name = vim_strsave(path_name);
-
-  name = saved_name;
-  while (*name) {
-    /* find the end of one dot-separated name and put a NUL at the dot */
-    p = menu_name_skip(name);
-
-    while (menu != NULL) {
-      if (menu_name_equal(name, menu)) {
-        if (menu->children == NULL) {
-          /* found a menu item instead of a sub-menu */
-          if (*p == NUL)
-            EMSG(_("E336: Menu path must lead to a sub-menu"));
```

Comment:
Thank you for the insight.

I m newbie to C, never used po files and english is not my native language so : what concretly mean "might be useful for fuzzy translations" ?